lib/bolt/util.rb

```diff
@@ -85,6 +85,44 @@ def first_run?
         Bolt::Config.user_path && !File.exist?(first_runs_free)
       end
 
+      def find_file_in_mod(mod, module_file)
```

Comment:
This method, `find_module_manually`, and `search_module` should have comments describing what the method does. While they're not the most complex methods, I think it's preferable to have a couple sentences describing something instead of requiring someone to parse the code.